src/ML-METATENSOR/pair_metatensor.h

```diff
@@ -0,0 +1,73 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
```

Comment:
```suggestion
   LAMMPS Development team: developers@lammps.org
```
Same for the other files.

Reply:
Sorry, that was a copy/paste leftover. I'll change it!